components/logged_in/logged_in.jsx

```diff
@@ -33,6 +33,8 @@ export default class LoggedIn extends React.Component {
         this.state = {
             user: UserStore.getCurrentUser(),
         };
+
+        document.getElementsByTagName('body')[0].className += ' channel-view';
```

Comment:
This does work but I have a couple of questions probably not entirely related to the PR. Let me know what you think.

Is there a specific reason for adding it to just logged in routes?. Because if we do navigate this route should we remove it? if not can we add to the body with css itself?.

Reply:
Not a specific reason for it. I've chosen a way to affect contextually because I was not 100% sure to add global style to <body>. But yes, to add CSS style will be fine.